website/data/api-navigation.js

```diff
@@ -111,6 +111,7 @@ export default [
       'internal-ui-mounts',
       'key-status',
       'leader',
+      'lease-count-quotas',
```

Comment:
I think on the actual website these should all be grouped together.

Reply:
`website/data/api-navigation.js` looks to be alphabetically ordered so I was just following convention. I can group these if my assumption is wrong.

Reply:
I did notice that, but I think the usability of the website should take precedence of the alphabetical ordering of the file? We could rename the file with some kind of prefix if we want to keep the ordering in this file alphabetical, or if there's a linter insisting on alphabetical order in the file.

Reply:
I don't think there is a linter? I'll group them 👍